src/cmd/check.go

```diff
@@ -2,28 +2,84 @@ package cmd
 
 import (
 	"fmt"
+	"github.com/godbus/dbus"
+	"github.com/nasz-elektryk/spito/api"
 	"github.com/nasz-elektryk/spito/checker"
+	cmdApi "github.com/nasz-elektryk/spito/cmd/cmdApi"
+	"github.com/nasz-elektryk/spito/cmd/guiApi"
 	"github.com/spf13/cobra"
 	"os"
 )
 
+var checkFileCmd = &cobra.Command{
+	Use:   "check file {path}",
+	Short: "Check local lua rule file",
+	Args:  cobra.ExactArgs(2),
+	Run: func(cmd *cobra.Command, args []string) {
+		runtimeData := getInitialRuntimeData(cmd)
+		script, err := os.ReadFile(args[1])
+		if err != nil {
+			panic(err)
```

Comment:
I don't trust it

Reply:
I change my mind about this particular panic and resolved it in the newest commit